schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+                    "enum": ["Javascript/Node.js"]
+                },
+                "title": {
+                    "description": "[DEPRECATED] Longer name of adapter to show in admin",
+                    "type": "string"
+                },
+                "titleLang": {
+                    "description": "Longer name of adapter in all supported languages like {en: 'Adapter', de: 'adapter', ru: 'Драйвер'}",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "news": {
+                    "description": "Short form of changelog in all supported languages",
+                    "type": "object",
+                    "minProperties": 1,
+                    "maxProperties": 20,
+                    "patternProperties": {
+                        "^\\d+\\.\\d+\\.\\d+.*": {
+                            "type": "object",
+                            "required": ["en"],
+                            "patternProperties": {
+                                "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                                    "type": "string"
+                                }
+                            },
+                            "additionalProperties": false
+                        }
+                     },
+                    "additionalProperties": false
+                },
+                "desc": {
+                    "description": "Description, object for multilingual description",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "messagebox": {
+                    "description": "true if messagebox supported. If yes, the object system.adapter.<instance>.messagebox will be created to send messages to adapter",
+                    "type": "boolean"
+                },
+                "readme": {
+                    "description": "[DEPRECATED]: Use docs",
+                    "type": "string"
+                },
+                "docs": {
+                    "description": "The structure like {\"en\": \"docs/en/README.md\", \"de\": [\"docs/de/README.md\", \"docs/de/README1.md\"]} that describes the documentation if not in README.md",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
```

Comment:
This can also be an array of strings

Reply:
could you provide an example? like multiple links for one's languages docs?

Reply:
Nevermind, you have it in the description. Line 86

Reply:
```
\"de\": [\"docs/de/README.md\", \"docs/de/README1.md\"]
```

Reply:
lol, thanks